### ISSUE TITLE
node install.js fails because it does not create path to kernel.json

### ISSUE THREAD

**Issue by io41** [notablemind/jupyter-nodejs#11]
```
$ node install.js
fs.js:500
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^
Error: ENOENT, no such file or directory '/Users/tim/.ipython/kernels/nodejs/kernel.json'
```

This is because only `/Users/tim/.ipython` exists. It should be doing an `mkdir -p` on the directory path (i.e. including the `keynels/nodejs` part) to kernel.json.

**Comment by themistymay** [comment]
Confirmed on my system as well.

**Comment by jaredly** [comment]
I just realized I didn't publish a new version after this was fixed. Sorry!
Fixed in 1.2.0
